core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpActivity.kt

```diff
@@ -39,7 +39,7 @@ class HelpActivity : BaseActivity() {
       R.string.help_5 to R.array.description_help_5
     ).associate { (title, description) ->
       getString(title) to resources.getStringArray(description)
-        .joinToString(separator = System.getProperty("line.separator")!!)
+        .joinToString(separator = "\n", postfix = "\n")
```

Comment:
Why this change?

Reply:
The HelpActivityTest is written to find view with text like this.

Reply:
Have you tried running the test locally after making this change? Did it pass?

Reply:
Yes I tried but they didn't pass. It couldn't find any views at all.

Reply:
Revert this and change the test code to use join to string, it putting a `/n` at the end is lazy anyway when joinToString is a much better way